pkg/server/test/list_objects.go

```diff
@@ -141,21 +201,6 @@ func ListObjectsTest(t *testing.T, ds storage.OpenFGADatastore) {
 				expectedResult: []string{},
 				expectedError:  nil,
 			},
-			{
-				name: "returns_error_if_duplicate_contextual_tuples",
```

Comment:
Removed this test and functionality associated with it. If the developer wants to provide a duplicate contextual tuple, then so be it. I don't see a reason why we should return an error if they provide a duplicate.

If they want to provide duplicates, then that should be fine so long as they still meet the maximum contextual tuple request limits.